lib/apartment/adapters/postgresql_adapter.rb

```diff
@@ -78,11 +75,16 @@ def connection_specification_name(config)
         if Apartment.pool_per_config
           "_apartment_#{config.hash}".to_sym
         else
-          host_hash = Digest::MD5.hexdigest(config[:host] || config[:url] || "127.0.0.1")
+          value = "#{config[:host]}:#{config[:port]}" || config[:url] || "127.0.0.1"
```

Comment:
Should `url` override `host/port` here to follow the rails pattern?

Reply:
Also wondering if there should be a `config[:host]` option like there is in the original code. That, of course, would make this a little more complicated to calculate.

Reply:
We end up setting `pool_per_config` to true, so these changes aren't relevant anymore.

Reply:
should we revert these changes then or is there a reason to keep them?

Reply:
It could potentially help us locally if we ever decided to turn off `pool_per_config`, since we run different postgres servers on different ports.